assets/styles.css

```diff
@@ -1,3 +1,409 @@
+  align-items: center;
+  cursor: pointer;
+}
+.socialIcons {
+  /* margin-left: 350px; */
+}
+.socialIcons svg {
+  background-color: #eee;
+  border: 1px #eee solid;
+  border-radius: 100%;
+  align-self: center;
+  vertical-align: middle;
+  padding: 5px;
+  width: 15px;
+  height: 15px;
+  /* width: 15px; */
+  /* height: 15px; */
+  /* max-width: fit-content; */
+  /* justify-self: center;
+  text-align: center; */
+}
+.bookmark_comment {
+  display: flex;
+  border-radius: 100%;
+  margin-top: 10px;
+  width: 15px;
+  height: 15px;
+}
+.bookmark_comment a i::before {
+  border-color: rgb(102, 102, 102) transparent;
+  border-style: solid;
+  bottom: -7px;
+  border-width: 6px 0px 0px 8px;
+  content: "";
+  display: block;
+  left: 15px;
+  position: absolute;
+  width: 0px;
+}
+.bookmark_comment a i {
+  height: 15px;
+  margin-top: 12px;
+}
+.bookmark_comment a i::after {
+  border-color: rgb(255, 255, 255) transparent;
+  border-style: solid;
+  border-width: 6px 0px 0px 6px;
+  bottom: -6px;
+  content: "";
+  display: block;
+  left: 16px;
+  position: absolute;
+  width: 0px;
+}
+.comment-box {
+  align-items: center;
+  background: transparent;
+  border: 1px solid rgb(102, 102, 102);
+  border-radius: 4px;
+  display: flex;
+  justify-content: center;
+  min-width: 26px;
+  min-height: 14px;
+  padding: 4px;
+  position: relative;
+  width: auto;
+}
+.hero-figure {
+  width: 90%;
+}
+.hero-figure img {
+  width: 80%;
+  padding-left: 150px;
+  padding-right: 150px;
+  cursor: pointer;
+}
+figcaption {
+  width: 60%;
+  padding-left: 145px;
+}
+figcaption span {
+  margin-right: 7px;
+  color: #666;
+  font-family: nyt-imperial, georgia, "times new roman", times, serif;
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+}
+
+.big-band p {
+  margin-bottom: 0.78125rem;
+  margin-top: 0;
+  font-family: nyt-imperial, georgia, "times new roman", times, serif;
+  font-size: 1.125rem;
+  line-height: 1.5625rem;
+  margin-left: 20px;
+  margin-right: 20px;
+  width: calc(100% - 40px);
+  max-width: 600px;
+  color: #333;
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+}
+footer {
+  padding-bottom: 20px;
+  border-top: 1px solid #ebebeb;
+  padding-top: 9px;
+  margin: 0 auto;
+  width: 1200px;
+  text-align: center;
+  font-size: 11px;
+}
+footer a {
+  padding-inline-start: 20px;
+  text-decoration: none;
+  color: #333;
+}
+.credits {
+  color: #888;
+  letter-spacing: 0.01em;
+  font-size: 0.8125rem;
+  font-family: nyt-imperial, georgia, "times new roman", times, serif;
+}
+.big-band {
+  width: 600px;
+  margin: 0 auto;
+}
+nav {
+  margin: 0 auto;
+  width: 1200px;
+  padding-bottom: 20px;
+  padding-top: 9px;
+  border-bottom: 1px solid #e2e2e2;
+  border-top: 2px solid #e2e2e2;
+  content: "";
+  display: block;
+}
+.nav-header {
+  display: flex;
+  flex-flow: row;
+  height: 45px;
+}
+.go-to-home {
+  margin: 18px 0px 0px auto;
```

Comment:
- [ ] Values of 0 shouldn't have units specified. Please remove px. with 0.

Reply:
Done.